fmriprep/cli/parser.py

```diff
@@ -378,11 +378,11 @@ def parse_args(args=None, namespace=None):
     # Wipe out existing work_dir
     if opts.clean_workdir and work_dir.exists():
         from niworkflows.utils.misc import clean_directory
-        build_log.log("Clearing previous fMRIPrep working directory: %s",
-                      work_dir)
+        build_log.info(f"Clearing previous fMRIPrep working directory: {str(work_dir)}")
```

Comment:
By default everything should be coerced to str.

```suggestion
        build_log.info(f"Clearing previous fMRIPrep working directory: {work_dir}")
```

e.g.:

```Python
>>> from pathlib import Path
>>> a = Path("/tmp")
>>> f"{a}"
'/tmp'
```

Reply:
good to know!